extension/src/background/helpers/account.ts

```diff
@@ -256,10 +256,6 @@ export const subscribeAccount = async (publicKey: string) => {
     }
   } catch (e) {
     console.error(e);
-    // Turn on when Mercury is enabled
```

Comment:
We shouldn't need these, we can get these logs from our server logs and probably don't want this triggering when Mercury is turned off.